tests/unit/core/test_installAddon.js

```diff
@@ -72,6 +72,9 @@ function componentWithInstallHelpers() {
 
 const defaultProps = ({
   _addonManager = getFakeAddonManagerWrapper(),
+  _clearConversionInfo = sinon.spy(),
+  _getConversionInfo = sinon.spy(),
+  _sendSponsoredEventBeacon = sinon.spy(),
```

Comment:
shouldn't we have `_trackConversion` instead?

Reply:
I dont see the new changes

Reply:
I think you might have checked before I pushed my latest commit. I marked it as fixed, but that just meant I had fixed it locally. The commit I pushed 11 minutes ago contains the changes. You're too fast for me!